src/main/java/dk/cngroup/lentils/entity/formEntity/CypherFormObject.java

```diff
@@ -96,6 +98,14 @@ public void setPlaceDescription(final String placeDescription) {
         this.placeDescription = placeDescription;
     }
 
+    public String getOrganizer() {
```

Comment:
Toto by bylo lepsi, kdyby bylo typu `User`, protoze ve skutecnosti se tam pak budou prirazovat uzivatele.

Reply:
Do budoucna souhlasím.
V UI potom nemít input field and select-box?

Akorát jsem přemýšlel, že momentálně nemáme u uživatele jméno, jen username. Tak pak by měli být co možná nejčitelnější.